### PR TITLE
pm4py 2.7.11.8 :snowflake:

### PR DESCRIPTION
**Pull request by lorepirri**
pm4py 2.7.11.8 :snowflake:

**Destination channel:** Snowflake

### Links

- [PKG-4246]
- dev_url: https://github.com/pm4py/pm4py-core/tree/2.7.11.8
- pypi: https://pypi.org/project/pm4py/2.7.11.8
- pypi inspector: https://inspector.pypi.io/project/pm4py/2.7.11.8

### Explanation of changes:

- new recipe
- upstream tests
- skip `win` and `s390x` because `cvxopt` is missing
- linter complains about `invalid_url` because their certificate has expired

[PKG-4246]: https://anaconda.atlassian.net/browse/PKG-4246?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  license_family: GPL
+  license_file: LICENSE
+  summary: Process mining for Python
+  description: |
+    pm4py is a python library that supports (state-of-the-art) process mining
+    algorithms in python. It is open source (licensed under GPL) and intended
+    to be used in both academia and industry projects. pm4py is a product of
+    the Fraunhofer Institute for Applied Information Technology.
+  doc_url: https://pm4py.fit.fraunhofer.de
+  dev_url: https://github.com/pm4py/pm4py-source
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
  dev_url: https://github.com/pm4py/pm4py-core
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,72 @@
+{% set name = "pm4py" %}
+{% set version = "2.7.11.8" %}
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:2]: What about the latest version https://github.com/pm4py/pm4py-core/tree/2.7.11.9?

File: recipe/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    - tests
+  commands:
+    - pip check
+    - cd tests
+    # as suggested in tests/README.txt
+    - python execute_tests.py
+
+about:
+  home: https://pm4py.fit.fraunhofer.de
+  license: GPL-3.0
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: `GPL-3.0-only` or `GPL-3.0-or-later`?

File: recipe/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    - wheel
+  run:
+    - python
+    - numpy <2
+    - pandas
+    - deprecation
+    - networkx
+    - python-graphviz
+    - wheel
+    - setuptools
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
```
They are for building the package only

File: recipe/meta.yaml (hunk 3 above)
Comment (lorepirri) [reply recipe/meta.yaml]: Nothing is specified, so I think it's `GPL-3.0-only`

```suggestion
  license: GPL-3.0-only
```

File: recipe/meta.yaml (hunk 2 above)
Comment (lorepirri) [reply recipe/meta.yaml:2]: We refined this version. The next one has not so many improvements, and they forgot to include `requirements.txt` in the tarball and the build fails. We'll fix it on the next time.

https://inspector.pypi.io/project/pm4py/2.7.11.9/packages/b4/57/f9939756a5fbc4225e4d787387e39009a047b965f12b9f25a627f7cc5299/pm4py-2.7.11.9.tar.gz/

Comment (skupr-anaconda) [reply recipe/meta.yaml:2]: Thanks!

Comment (lorepirri) [reply recipe/meta.yaml:2]: Thank you!

**Review by skupr-anaconda — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (lorepirri) [reply recipe/meta.yaml]: done

File: recipe/meta.yaml (hunk 3 above)
Comment (lorepirri) [reply recipe/meta.yaml]: done, thank you for noticing it

**Review by psteyer — APPROVED** [review]

**Pull request merged**